### ISSUE TITLE
Sprint 44 (4/11-4/24)

### ISSUE THREAD

**Issue by skirkman16** [department-of-veterans-affairs/va.gov-team#80528]
## PTO
Mateo: 4/12
Carl: 4/15
Phil: 4/18 (can respond via Slack), 4/19 (OOO)
Anu: 4/19 (async only), 4/22-4/24
Josie: 4/24

## Sprint 44 Goals

**Campaigns**
_High-level goal: Prepare for behavior change communications campaign pause lifting._ 

Engineering:
- USiP PR, launch (blocked until SSiT OCTO gives word)

Design/Research:
- Deploy GovDelivery segmented emails (blocked, may unblock this sprint)

Data:
- Update dashboard with support and helpdesk data
- Monitor sign in and call center data (blocked until campaigns resume)
- Automate MPI, SSOe, MHV data ingest (icebox)

**Stretch Goals**

**IPP**
_High level goal: Prepare for IPP Service Pilot 1_

Engineering: Launch IPP Applications in LHDI DEV/QA regions.
- Start VASI scan, eMASS, WASA, PTA security countermeasures for LHDI
- Deploy IPP app to LHDI DEV/QA regions
- Get Datadog access for IPP logging and monitoring
- Start on Section 4 Development Countermeasures
- Start on Section 3 Architecture Countermeasures

Research/Design: 

- Identify potential verifiers for pilot
- Make final content changes to IPP application
- Complete design refinements for application for pilot (remove accordions)
- Create pop up page between sign-in page for Veteran facing application and start of IPP flow
- A11y updates - design (Phil; deadline: Friday 9AM EST)

**Stretch Goals**
- Onboard to Datadog
- Create a landing page for proofing agent facing application
- Walk in onboarding flow (flow and blueprint only)
- Reschedule appointment flow (flow and blueprint only)

## What we want to demo at the end of the sprint

- Updated dashboard w/ call center and helpdesk data + MHV dashboard
- LHDI deployment
- Screenshot or load up SD Elements to show progress or highlight an interesting topic (maybe)
- Final content and design changes for pilot 1 w/ a11y updates (before and after pictures)
